libs/application/templates/children-residence-change/src/dataProviders/MockChildrenNationalRegistryProvider.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Application,
+  BasicDataProvider,
+  FailedDataProviderResult,
+  SuccessfulDataProviderResult,
+} from '@island.is/application/core'
+import { CRCApplication } from '../types'
+
+import { PersonResidenceChange } from '@island.is/application/templates/children-residence-change'
+
+/** This is a temporary mock provider for children residence change. National registry team are setting up real provider which we will connect to once it is up. **/
+export class MockChildrenNationalRegistryProvider extends BasicDataProvider {
+  readonly type = 'MockChildrenNationalRegistry'
```

Comment:
More a general question but should we declare the type on a higher level and export and then use that in the form, instead of declaring the keys twice?

Reply:
Are we declaring the key twice? One is `MockChildrenNationalRegistry` and the other is `ChildrenNationalRegistry`

Reply:
I am talking about this line and this one [here](https://github.com/island-is/island.is/pull/3287/files#diff-f6720ae4eb7c800bc3673af128003c2cc1b630cd096474f1f36a7c9b594994eeR131)

Reply:
Ahh true, I will change this and for all the other providers also